tests/test_parsers.py

```diff
@@ -78,3 +78,29 @@ def test_config_t():
     assert config.run and config.model
 
     shutil.rmtree(log_dir)
+
+
+
+# not a unit test, more of an integration test
+@pytest.mark.skipif(
+    platform.system() != "Linux", reason="FIXME: Test setup is Linux specific"
+)
+def test_transformations():
+    conf_dir = Path("tests/conf")
+    rules = read_yaml(conf_dir / "transform_rules.yaml")
+    # remove validator because none are implemented
+    conf_rules = remap(rules, visit=lambda p, k, v: k not in ("validator",))
+    config_t = get_config_t(conf_rules)
+
+    # ensure dirs/files exist
+    log_dir = Path("/tmp/pydc-dir")
+    log_dir.mkdir(exist_ok=True)
+    (log_dir / "transform_file.log").touch()
+
+    config = config_t.from_yaml(conf_dir / "transform_config.yaml")
+    assert config.run and config.model
+
+    shutil.rmtree(log_dir)
+
+
+test_transformations()
```

Comment:
```suggestion

```
This whole block shouldn't be there.  You also created a new rules file (which hasn't been committed actually), that's also not right.  The idea is, updating the rules should let the user generate new config, there shouldn't be a need to update the code!  So all the old tests should suffice, just you need to add a transformation in one of rules in the existing `rules.yaml`.